pkg/migrations/sql/versions/dev/0.7.0-beta.1.dev/1-tracing.sql

```diff
@@ -78,4 +78,5 @@ $block$
 ;
 
 DROP FUNCTION IF EXISTS SCHEMA_TRACING_PUBLIC.put_tag_key(SCHEMA_TRACING_PUBLIC.tag_k, SCHEMA_TRACING_PUBLIC.tag_type);
-DROP FUNCTION IF EXISTS SCHEMA_TRACING_PUBLIC.put_tag(SCHEMA_TRACING_PUBLIC.tag_k, SCHEMA_TRACING_PUBLIC.tag_v, SCHEMA_TRACING_PUBLIC.tag_type);
\ No newline at end of file
+DROP FUNCTION IF EXISTS SCHEMA_TRACING_PUBLIC.put_tag(SCHEMA_TRACING_PUBLIC.tag_k, SCHEMA_TRACING_PUBLIC.tag_v, SCHEMA_TRACING_PUBLIC.tag_type);
```

Comment:
I have a question regarding conventions of these "versioned" migration files. `0.7.0-beta.1` has already been released. Does that mean that this file contains all changes _since_ `0.7.0-beta.1`?

Reply:
Good point.

>Does that mean that this file contains all changes since 0.7.0-beta.1?

I think the answer to that is "no".

Reply:
Does that mean that we need to create a new `0.7.0-beta.2.dev` dir? Or `0.7.0`?

Reply:
@cevian I noticed that you have the same thing going on in #892.

Reply:
Okay, I'm going to follow the pattern from #892

Reply:
The version for this file is actually the correct `0.7.0-beta.1.dev.1` which comes after  `0.7.0-beta.1` according to semver. During development we don't know if we'll release `0.7.0-beta.2` or `0.7.0` So we bump versions as little as possible. See https://github.com/timescale/promscale/blob/master/pkg/version/version.go#L16

Reply:
I'd prefer not to modify existing upgrade scripts though, instead only adding new scripts in PRs. So let's move to another script file? 2-blah?

Reply:
done.